applications/serial_lte_modem/sample.yaml

```diff
@@ -1,15 +1,15 @@
 sample:
-  name: serial lte modem sample
+  name: Serial LTE Modem application
 tests:
-  applications.nrf9160.serial_lte_modem:
```

Comment:
Do you also want to remove nRF9160 from the name of the app in the docs?
I see 9161 is in platform_allow, so having nRF9160 in the app name might be a thing outdated.
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/README.html

Reply:
I'll rename this to `nRF91 Series`.

Reply:
Ah, but #12120 uses just "Serial LTE modem", so I think @eivindj-nordic that "nRF91 Series" in the doc title is not needed.